app/helpers/README.md

```diff
@@ -0,0 +1,20 @@
+# Import Pipeline Indoor Map
+
+### Labeling
+- The first step is labeling the image of a room plan. For this you can use the labeling tool https://www.makesense.ai
+- For every corner of a room insert a point
+- Create a label for each point
+- Label Fomat
+    - Every room is identified by a letter, the number after the letter is the position of a point in the surrounding polygon
+    - A point can be part of a maximum of four rooms. This is why a label is made up of four parts, seperated by commas.
```

Comment:
I would disagree. In a pizza, the center point is part of all the slices. Simultaneously, a point could be a corner point of any number of rooms.

Reply:
Do you know a room at the hpi, where this occurs ? 
I mean you are right. But if this case never happens we can ignore this.

Reply:
Fair enough, then the comment should be changed though. This sounds like the pizza case is impossible, when you are just choosing to ignore it. Maybe `"For now, we are only modeling scenarios where a point can be part of up to 4 rooms. They are defined by the four parts, separated by commas. If a point should be part of more than 4 rooms, the script should be adapted"`.?

Reply:
Ok, I'll change that!

Reply:
gonna add that